--- a/src/AudioStream.cpp
+++ b/src/AudioStream.cpp
@@ -389,6 +389,6 @@
-        av_opt_set_chlayout  (m_swrCtx, "in_channel_layout",  &m_codec_context->ch_layout,     0);
+        av_opt_set_chlayout  (m_swrCtx, "in_chlayout",         &m_codec_context->ch_layout,     0);
         av_opt_set_int       (m_swrCtx, "in_sample_rate",     m_codec_context->sample_rate,    0);
         av_opt_set_sample_fmt(m_swrCtx, "in_sample_fmt",      m_codec_context->sample_fmt,     0);
-        av_opt_set_chlayout  (m_swrCtx, "out_channel_layout", &layoutOut,                      0);
+        av_opt_set_chlayout  (m_swrCtx, "out_chlayout",        &layoutOut,                      0);
         av_opt_set_int       (m_swrCtx, "out_sample_rate",    m_codec_context->sample_rate,    0);
         av_opt_set_sample_fmt(m_swrCtx, "out_sample_fmt",     AV_SAMPLE_FMT_S16,               0);
